### ISSUE TITLE
Plugin Error and Crash

### ISSUE THREAD

**Issue by GETASFAR** [synfig/synfig#502]
**Synfig version & platform**:
Synfig Version: 1.3.6 (Development Snapshot)
Platform: Windows 10 Pro, 64bit

**Issue description**:
Whenever I try to use the plug-ins (like "add Skeleton (Simple)"), I receive these strange errors (please check attachments) and then Synfig crashes. A similar error and crash happens with the "Unhide all layers" plugin too. I have uninstalled and reinstalled Synfig with no change in error. 

![capture1 mod](https://user-images.githubusercontent.com/37457158/37543211-3c4ba5d8-2937-11e8-9e24-b4a8fac0c88a.PNG)
![capture2 mod](https://user-images.githubusercontent.com/37457158/37543213-3e41543c-2937-11e8-8ace-0bcae078cc0e.PNG)

**Comment by morevnaproject** [comment]
Hello! Please try to save file before you call plugin. Does that helps?

**Comment by GETASFAR** [comment]
I have tried saving the project and then calling the plugin with no change in error. I remember being able to use plugins in a previous version of Synfig, so I uninstalled 1.3.6 and installed 1.3.5; however, the error still occurred. I have noticed that if I have multiple projects up and try calling a plugin, only the project that is trying to use the plugin closes. If it is the only project up when I call the plugin, then the project closes and Synfig crashes. I forgot to mention that I am using the 64-bit version of Synfig. I installed the 32-bit version of Synfig but I still experienced the error. I have noticed that the first window's final line "Synfig Animation 1.48CF5A66.sif" changes every error. The "48CF5A66" part of the error has a different string of characters anytime I try calling the plugin.
